exercises/practice/acronym/.meta/zcl_acronym_example.clas.abap

```diff
@@ -0,0 +1,29 @@
+CLASS zcl_acronym_example DEFINITION
```

Comment:
try renaming this class to `zcl_acronym`

during CI it is checked that this solution works with the unit tests

Reply:
thanks, i renamed it.